src/buildCommon.js

```diff
@@ -367,6 +367,16 @@ var makeVList = function(children, positionType, positionData, options) {
     return vlist;
 };
 
+// Converts verb group into body string, dealing with \verb* form
+var makeVerb = function(group, options) {
+    var text = group.value.body;
+    if (group.value.star) {
+        text = text.replace(/ /g, '\u2423');  // Open box
```

Comment:
It seems that the symbol is the wrong size, so probably not in the font...  I thought that the usual "no metrics" warning would trigger in this case, but it seems not to, because I'm passing an entire string instead of individual characters.  I changed the main code to look up characters one at a time, then try to combine them, as in `text`.